apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/Trace/SimpleManagedTracerTest.cs

```diff
@@ -73,10 +77,86 @@ private static bool IsValidSpan(TraceSpan span, string name, ulong parentId, Spa
         /// a difference of 1ms between start and end. If we do not the Trace
         /// API will not record the span.
         /// </summary>
-        private static bool Is1MsSpan(TraceSpan span)
+        private static bool IsAtLeast1MsSpan(TraceSpan span)
         {
             var duration = span.EndTime - span.StartTime;
-            return duration.Seconds == 0 && duration.Nanos == 1_000_000;
+            return duration.Seconds >= 1 || duration.Nanos >= 1_000_000;
```

Comment:
To be absolutely sure, probably

```csharp
duration.Seconds >= 1 || (duration.Seconds == 0 && duration.Nanos >= 1_000_000);
```

Otherwise if we had our start and end time the wrong way round we'd allow a negative value to pass.

Reply:
Yep, will change and add a comment or else that second check on seconds looks redundant.